styles/canary.json

```diff
@@ -52,19 +52,20 @@
 					"slug": "small"
 				},
 				{
-					"size": "0.75rem",
+					"size": "1.125rem",
```

Comment:
If the only change to the fontSizes compared to theme.json is that they are not fluid, I wonder if this could be simplified 🤔

Reply:
The difference with the Canary fontSizes is that they're all the same size.